terminal.py

```diff
@@ -147,6 +147,8 @@ def print_help(self):
         mt.add_menu("portfolio")
         mt.add_menu("dashboards")
         mt.add_menu("reports")
+        mt.add_raw("\n")
+        mt.add_cmd("wiki")
```

Comment:
Let's add this just below update, ok?

Reply:
Thank you for the feedback!! Just committed the adjustments